components/workouts/CreateWorkout.jsx

```diff
@@ -0,0 +1,48 @@
+// eslint-disable-next-line no-unused-vars
+import * as React from 'react';
+import { View } from 'react-native';
+import styled from 'styled-components/native';
+import FinishButton from '../utils/FinishButton';
+import PlusButton from '../utils/PlusButton';
+
+const TitleTextInput = styled.TextInput`
+  font-family: 'Montserrat_600SemiBold';
+  font-size: 24px;
+  margin: 15px 3%;
+  width: 67%;
+  borderBottomWidth: 1px;
+`;
+
+const AddFinishButton = styled(FinishButton)`
+  position: absolute;
+  top: 20px;
+  right: 20px;
+`;
+
+const AddCycleButton = styled(PlusButton)`
+   position: absolute;
+   bottom: 20px;
+   right: 25px;
+`;
+
+export default () => {
+  const [name, setName] = React.useState('Set Workout Name');
+
+  return (
+    <View style={{ height: '100%' }}>
+      <View>
+        <TitleTextInput
+          onChangeText={() => setName(name)}
+          value={name}
+          maxLength={40}
```

Comment:
Should we make this ~15 chars, I think that's about how many will fit on two lines of the workout cards